app/controllers/v2/content_items_controller.rb

```diff
@@ -12,6 +14,18 @@ def index
       ).call
     end
 
+    def linkables
+      render json: Queries::GetContentCollection.new(
+        content_format: request.query_parameters.fetch(:format),
+        fields: %w(
+          title
+          content_id
+          links
+          publication_state
```

Comment:
How many use cases do we think this endpoint will be used for? Will that list of fields cover those use cases or do we need to think about making it configurable?